libs/api-client-bear/src/user.ts

```diff
@@ -188,25 +156,23 @@ export class UserModule {
     /**
      * Returns info about currently logged in user from bootstrap resource
      */
-    public getAccountInfo(): Promise<{
+    public async getAccountInfo(): Promise<{
         login: string;
         loginMD5: string;
         firstName: string;
         lastName: string;
         organizationName: string;
         profileUri: string;
     }> {
-        return this.xhr.get("/gdc/app/account/bootstrap").then((result: any) => {
-            const { bootstrapResource } = result.getData();
-            return {
-                login: bootstrapResource.accountSetting.login,
-                loginMD5: bootstrapResource.current.loginMD5,
-                firstName: bootstrapResource.accountSetting.firstName,
-                lastName: bootstrapResource.accountSetting.lastName,
-                organizationName: bootstrapResource.settings.organizationName,
-                profileUri: bootstrapResource.accountSetting.links.self,
-            };
-        });
+        const { bootstrapResource } = await this.getBootstrapResource();
```

Comment:
This change can imo cause some issues, I remember that sometimes, binding xhr module to some methods of the bear-client was necessary to make it work.